src/Process/Listener/MutationTestingResultsLoggerSubscriber.php

```diff
@@ -92,7 +92,7 @@ public function onMutationTestingFinished(MutationTestingFinished $event)
     private function filterLogTypes(array $logTypes): array
     {
         foreach ($logTypes as $key => $value) {
-            if ($this->logVerbosity === LogVerbosity::NONE) {
+            if ($this->logVerbosity === LogVerbosity::NONE || (int) $this->logVerbosity === LogVerbosity::NONE_INTEGER) {
```

Comment:
Would it be better if we convert a numeric verbosity to a string in the beginning, and avoid using deprecated constants in other code?